src/MooseSoft.Azure.ServiceBus/BackOffDelayStrategy/LinearBackOffDelayStrategy.cs

```diff
@@ -8,19 +8,19 @@ namespace MooseSoft.Azure.ServiceBus.BackOffDelayStrategy
     /// </summary>
     public class LinearBackOffDelayStrategy : IBackOffDelayStrategy
     {
-        private readonly int _multiplier;
+        private readonly TimeSpan _multiplier;
 
-        public LinearBackOffDelayStrategy(int multiplier)
+        public LinearBackOffDelayStrategy(TimeSpan multiplier)
         {
             _multiplier = multiplier;
         }
 
```

Comment:
A multiplier as a time span really doesn't make sense.  The idea here was to set a multiplier that will scale up by the number of attempts.  The name might not be the best for this but I'd remove these changes.

Reply:
reverted these changes.